iroh-one/Cargo.toml

```diff
@@ -40,6 +40,6 @@ http = "0.2"
 default = ["rpc-mem", "rpc-grpc"]
 rpc-grpc = ["iroh-rpc-types/grpc", "iroh-rpc-client/grpc", "iroh-metrics/rpc-grpc"]
 rpc-mem = ["iroh-rpc-types/mem", "iroh-rpc-client/mem"]
-# uds-gateway controls whether http over uds is enabled. This is independent of the
+# http-uds-gateway controls whether http over uds is enabled. This is independent of the
 # rpc-grpc control endpoint.
```

Comment:
I'd move this entirely to the readme, it is relevant for users of the crate plus having comments here makes the dependencies harder to sort automatically.

Speaking of which, `http-ud-gateway` sorts differently than `uds-gateway` so should move up (this sorting was added to DEVELOPERS.md not too long ago).